app/views/propertyLinking/uploadEvidence.scala.html

```diff
@@ -125,9 +129,9 @@ <h1 class="govuk-heading-l">@messages("uploadOtherEvidence.title")</h1>
         ))
     }
 
-    <div id="file-upload-form" class="govuk-visually-hidden">
+    <div id="file-upload-form" class="govuk-!-display-none">
         @formWithCSRF(action = UploadController.initiate(EvidenceChoices.OTHER), 'enctype -> "multipart/form-data", 'id -> "uploadForm") {
-            <div class="govuk-visually-hidden" id="initiateFields"></div>
+            <div id="initiateFields" class="govuk-!-display-none"></div>
```

Comment:
I think we all following html Ids pattern ='abc-edf-xyz' rather camel case

Reply:
That `id` is already there and I didn't want to change it as it may be referenced in javascript.

Reply:
no problem , I got it